hivemq-edge/src/main/java/com/hivemq/edge/modules/config/impl/AbstractProtocolAdapterConfig.java

```diff
@@ -69,22 +69,22 @@ public enum MessageHandlingOptions {
                            defaultValue = "0")
         private int qos = 0;
 
-        @JsonProperty(value = "messageHandlingOptions", required = true)
+        @JsonProperty(value = "messageHandlingOptions", required = false)
         @ModuleConfigField(title = "Message Handling Options",
                            description = "The topic to publish data on",
```

Comment:
not really your change, but just noticed tat the description is not correct here.

```suggestion
                           description = "This setting defines the format of the resulting MQTT message, either a message per changed tag or a message per subscription that may include multiple data points per sample",
```